src/tests/Microsoft.PowerFx.Core.Tests/NamedFormulasTests.cs

```diff
@@ -28,12 +28,41 @@ public void DefFuncTest(string script)
         }
 
         [Theory]
-        [InlineData("Foo(x As Number, x As String) As Number = Abs(x);")]
-        public void DoubleArgDefTest(string script)
+        [InlineData("Rec4(x: Number): Number => { { force: 1, goo: x } };" +
```

Comment:
also some tests with newlines of various kinds, comments between udfs, before/after args, excessive whitespace, etc..